cmd/controller-manager/app/controller_manager.go

```diff
@@ -300,7 +300,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		}
 
 		glog.V(5).Info("Running controller")
-		go serviceCatalogController.Run(stop)
+		go serviceCatalogController.Run(int(s.ConcurrentSyncs), stop)
```

Comment:
elsewhere in the kube code, the number of workers is passed in this way

https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L143